model/textparse/openmetricsparse.go

```diff
@@ -351,7 +461,15 @@ func (p *OpenMetricsParser) Next() (Entry, error) {
 			p.series = p.l.b[p.start:p.l.i]
 			t2 = p.nextToken()
 		}
-		return p.parseMetricSuffix(t2)
+
+		suffixEntry, err := p.parseMetricSuffix(t2)
+		if err != nil {
+			return suffixEntry, err
+		}
+		if p.skipCTSeries && p.isCreatedSeries() {
+			return p.Next()
+		}
+		return suffixEntry, err
```

Comment:
```suggestion
		return suffixEntry, nil
```

Reply:
covered in this week's sync up (including below review)